### PR TITLE
Make Travis check the master and branches

### PR DESCRIPTION
**Pull request by kalabiyau**
🥔

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+language: ruby
+rvm:
+  # The version used at SLE12
+  - 2.1.2
```
Comment (ikapelyukhin) [inline .travis.yml]: Hmm, which version are we aiming at? The RPM package is build for Ruby 2.4.

File: .travis.yml [hunk 2]
```diff
@@ -13,4 +13,3 @@ services:
 before_script:
   - psql -c "CREATE DATABASE potato;" -U postgres
   - psql -c "CREATE USER potato WITH PASSWORD 'potato';" -U postgres
```
Comment (ikapelyukhin) [inline .travis.yml]: `grant all privileges on database potato to potato;`?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+language: ruby
+rvm:
+  # SLES15 sp0
+  - 2.4.1
+
+  # Future versions
+  - ruby-head
+script:
+  - 'bundle exec rspec spec'
+  - 'bundle exec rubocop'
```
Comment (thutterer) [inline .travis.yml]: Does Travis stop and report after it hit the first error? If yes, I would move rubocop one line up (before rspec) as rubocop usually fails faster than a large number of rspec tests.
Could save us some time.

Comment (ikapelyukhin) [inline .travis.yml]: Maybe rubocop should be a separate Travis task?

Comment (ikapelyukhin) [inline .travis.yml]: It seems like Travis runs rubocop even if rspec failed: https://travis-ci.org/SUSE/rmt/jobs/258048380

Comment (thutterer) [inline .travis.yml]: Hm okay then it doesn't really matter.

**Pull request merged**